### ISSUE TITLE
pre-commit-config broken

### ISSUE THREAD

**Issue by AshirRashid** [intelowlproject/IntelOwl#1403]
## What happened
pre-commit-config uses Flake8 via GitLab but broke due to [Flake8 being taken down from GitLab](https://www.reddit.com/r/Python/comments/yvfww8/flake8_took_down_the_gitlab_repository_in_favor/).

## Environment
1. OS: MacOS 12.6
2. IntelOwl version: 4.1.4

##  What did you expect to happen
pre-commit to run flake8

## How to reproduce your issue
Making any commit with the pre-commit setup according to https://intelowl.readthedocs.io/en/latest/Contribute.html#create-a-pull-request

## Error messages and logs
```
remote: The project you were looking for could not be found or you don't have permission to view it.
fatal: repository 'https://gitlab.com/pycqa/flake8.git/' not found
```

**Comment by AshirRashid** [comment]
Can be easily fixed by changing 
`- repo: https://gitlab.com/PyCQA/flake8`
to
`- repo: https://github.com/PyCQA/flake8`

And updating the documentation.
I want to work on fixing it.

**Comment by AshirRashid** [comment]
There’s a slight error in the [IntelOwl](https://github.com/intelowlproject/IntelOwl/tree/develop)/[docs](https://github.com/intelowlproject/IntelOwl/tree/develop/docs)/[source](https://github.com/intelowlproject/IntelOwl/tree/develop/docs/source)/Contribute.md: a missing `.`.
`isort --profile black --filter-files --skip venv`
Correction:
`isort . --profile black --filter-files --skip venv`

Can I just correct it with this issue or should I open another issue for it?

**Comment by mlodic** [comment]
it would be great if you can fix them both in the same PR!

**Comment by mlodic** [comment]
solved in #1405
